app/src/utils/player.js

```diff
@@ -23,3 +23,13 @@ export function getOfficialHiscoresUrl(player) {
 
   return `https://secure.runescape.com/m=hiscore_oldschool${suffix}/hiscorepersonal.ws?user1=${username}`;
 }
+
+export function getTooltip(type) {
```

Comment:
Could you rename this to `getPlayerTooltip` please? As it's referenced as "getTooltip" anywhere on the app, it's a bit ambiguous.

Reply:
Will do